### PR TITLE
Delaying filter loading until after the activities are loaded.

### PR DESCRIPTION
**Pull request by cbusmand2l**

### CODE REVIEW

**Comment by cbusmand2l** [comment]
@AlexBedley Ah! I forgot that the skeleton state was based on whether the filters were loading or not. On that note, would it also make sense to remove it from the search summary state as well?

**Comment by maboelsoudd2l** [comment]
To add to what @AlexBedley said i captured this screenshot (through the magic of browser network throttling), which shows the filters loaded and the entity not-loaded. but that could also be due to the skeleton as you guys said.

image

**Comment by maboelsoudd2l** [comment]
The entangling of `filtersLoading` with the loading skeleton was something I added a while ago (probably because I thought that `filtersLoading` was related to filtering the data).

File: components/d2l-quick-eval/d2l-quick-eval-submissions.js [hunk 1]
```diff
@@ -532,12 +532,12 @@ class D2LQuickEvalSubmissions extends mixinBehaviors(
 		return _loadingMore;
 	}
 
-	_computeShowLoadingSkeleton(_loading, filtersLoading, searchLoading) {
-		return _loading || filtersLoading || searchLoading;
+	_computeShowLoadingSkeleton(_loading, searchLoading) {
+		return _loading || searchLoading;
```
Comment (maboelsoudd2l) [inline components/d2l-quick-eval/d2l-quick-eval-submissions.js]: If we're removing `filtersLoading`, by that logic should we also be removing `searchLoading` as well?

Comment (cbusmand2l) [reply components/d2l-quick-eval/d2l-quick-eval-submissions.js]: I don't know that the search has near as much overhead as the filters. We may want to just double check whether it's something we want to do, but either way I don't see it as a reason to hold up this PR.

Comment (maboelsoudd2l) [reply components/d2l-quick-eval/d2l-quick-eval-submissions.js]: fair enough

Comment (AlexBedley) [reply components/d2l-quick-eval/d2l-quick-eval-submissions.js]: Search doesn't really have loading associated with it, it was just to make the UI a bit snappier so it immediately displays the skeleton when you press enter

**Pull request merged**